core/modules/parsers/parseutils.js

```diff
@@ -208,6 +215,29 @@ exports.parseMacroInvocation = function(source,pos) {
 	return node;
 };
 
+exports.parseFilterVariable = function(source) {
+	var node = {
+			name: "",
+			params: [],
+		},
+		pos = 0,
+		reName = /([^\s>"'=]+)/g;
```

Comment:
Why do we use `=` as a terminator?

Reply:
@Jermolene I re-used the regular expression from [parseMacroInvocation](https://github.com/Jermolene/TiddlyWiki5/blob/master/core/modules/parsers/parseutils.js#L174) and I spotted this but left it alone as I wasn't completely sure why we had it in that context either. Will revise.